src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/Generated/UsageTestsGenerated.ps1

```diff
@@ -0,0 +1,46 @@
+﻿# <auto-generated>
+# Copyright (c) Microsoft and contributors.  All rights reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# 
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# 
+# 
+# Warning: This code was generated by a tool.
+# 
+# Changes to this file may cause incorrect behavior and will be lost if the
+# code is regenerated.
+# 
+# For documentation on code generator please visit
+#   https://aka.ms/nrp-code-generation
+# Please contact wanrpdev@microsoft.com if you need to make changes to this file.
+# </auto-generated>
+
+<#
+.SYNOPSIS
+Test creating new Usage using minimal set of parameters
+#>
+function Test-UsageCRUDMinimalParameters
+{
+    # Setup
+    $rname = Get-ResourceName;
+    $location = "westus";
```

Comment:
Same comment about location here

Reply:
`Get-ProviderLocation` [1] by default returns `West US` which fails validation on SDK level because of regexp in Swagger specs [2]. That's why we have to specify certain location for this test.

[1] https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/Common.ps1#L56

[2] https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2018-04-01/usage.json#L51

Reply:
Please file a bug to fix this - all cmdlets should accept both the display name as well as the shortened name of locations.  For this PR we can leave this.